packages/ember-metal/lib/meta.js

```diff
@@ -8,6 +8,16 @@ import EmptyObject from './empty_object';
 import { lookupDescriptor } from './utils';
 import symbol from './symbol';
 
+let metaCounters = window._metaCounters = {
```

Comment:
Need to remove this before landing, I've been using these counters to help try different variations...